test/features/sdk_flow.feature

```diff
@@ -154,3 +154,15 @@ Feature: SDK File Upload Tests
       | locale |
       |        |
       | es     |
+
+  Scenario Outline: I should be able to open, close and open the sdk again as a modal
+    Given I navigate to the SDK as a modal
+    When I click on modal_button (SDK)
+    Then I should see page_title ()
+    When I press esc key
+    When I click on modal_button ()
+    Then I should see page_title ()
+
+    Examples:
```

Comment:
since you're not using the examples, you can remove `Examples` and change `Scenario Outline` to `Scenario`. Default behaviour should default to English (I hope)

Reply:
Yes, it will default to English